test/tests/materials/lr_isolator/lr_isolator_PDelta.i

```diff
@@ -0,0 +1,477 @@
+# Test for lead rubber isolator
+# Axial direction
+
+[Mesh]
+  type = GeneratedMesh
+  xmin = 0
+  xmax = 1
+  nx = 1
+  dim = 1
+  displacements = 'disp_x disp_y disp_z'
+[]
+
+[Variables]
+  [./disp_x]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+  [./disp_y]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+  [./disp_z]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+  [./rot_x]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+  [./rot_y]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+  [./rot_z]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+[]
+
+[AuxVariables]
+  [./vel_x]
+  order = FIRST
+  family = LAGRANGE
+  [../]
+  [./vel_y]
+  order = FIRST
+  family = LAGRANGE
+  [../]
+  [./vel_z]
+  order = FIRST
+  family = LAGRANGE
+  [../]
+  [./accel_x]
+  order = FIRST
+  family = LAGRANGE
+  [../]
+  [./accel_y]
+  order = FIRST
+  family = LAGRANGE
+  [../]
+  [./accel_z]
+  order = FIRST
+  family = LAGRANGE
+  [../]
+  [./rot_vel_x]
+  order = FIRST
+  family = LAGRANGE
+  [../]
+  [./rot_vel_y]
+  order = FIRST
+  family = LAGRANGE
```

Comment:
Are the rotational velocities and accelerations required? It looks like they are not being used by any of the material classes.

Reply:
No, they are not. I will remove them.